cryptoshred-core/src/test/java/eu/prismacapacity/cryptoshred/core/metrics/CryptoMetricsBaseTest.java

```diff
@@ -6,7 +6,7 @@
 
 public class CryptoMetricsBaseTest {
 
-	CryptoMetrics.Base uut = new CryptoMetrics.Base() {
+	final CryptoMetrics.Base uut = new CryptoMetrics.Base() {
```

Comment:
suggestion: can we please agree to not do that on the test scope automatically?
i personally like the test scope using as little words as possible in order to make it easy to read.

Reply:
I see. need to tell this IDEA then. but I get your point. let it be this then:
```suggestion
	CryptoMetrics.Base uut = new CryptoMetrics.Base() {
```

Reply:
you can scope inspections in idea, which is actually a great feature that eclipse is lacking